src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -455,7 +455,7 @@ export default {
         // build filing item
         const item: any = {
           filingType,
-          title: `${this.entityTypeToName(this.entityType)} ${name} - ${this.entityName}`,
+          title: `${this.entityTypeToName(this.entityType)} ${name} - ${this.corpDisplayName}`,
```

Comment:
Is this change also needed on line 749 (ie, fetch/download receipt)?
You should be able to test this with a Paid IA.
Please test both Named and Numbered IAs.

Reply:
you are correct, just tried now